CPP_STYLE_GUIDE.md

```diff
@@ -200,6 +201,42 @@ void FooConsumer(std::unique_ptr<Foo> ptr);
 
 Never use `std::auto_ptr`. Instead, use `std::unique_ptr`.
 
+### Avoid non-const references
+
+Using non-const references often obscures which values are changed by an
```

Comment:
I suggest for the status quo

```suggestion
Using non-const references can obscures which values are changed by an
assignment. Consider using a pointer instead as it requires explicit
dereferencing.
```

And if possible add a reference to an outside source.

Reply:
> can obscures

"can obscure"?

Reply:
ack

Reply:
I’ve changed it to

> Using non-const references often obscures which values are changed by an
assignment. Consider using a pointer instead, which requires more explicit
syntax to indicate that modifications take place.

(In particular because the dereferencing part of the pointer syntax is not the only part we care about.)

Reply:
👍 your phasing is better (I'll compromise of the offten / can wording)